trollflow2/launcher.py

```diff
@@ -438,3 +440,61 @@ def sendmail(config, trace):
     pid = Popen([sendmail, "-t", "-oi"], stdin=PIPE)
     pid.communicate(msg.as_bytes())
     pid.terminate()
+
+
+def launch(args_in):
+    """Launch the processing."""
+    args = parse_args(args_in)
+
+    log_config = args.pop("log_config", None)
+    if log_config is not None:
+        with open(log_config) as fd:
+            log_config = yaml.safe_load(fd.read())
+
+    logger = logging.getLogger("satpy_launcher")
+
+    log_queue = Manager().Queue()
+
+    with logging_on(log_queue, log_config):
+        logger.warning("Launching Satpy-based runner.")
```

Comment:
Why is this a warning? This would trigger a alarm at our place...

Reply:
Oops, fixing to `.info()`.

Reply:
Fixed in https://github.com/pytroll/trollflow2/pull/159/commits/7bfb65180c8d5dda8cd6449757d9966edb3cbabc